### ISSUE TITLE
All global message notifications should be vocalized as soon as they happen

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13677]
**Describe the bug**
- Consistently and all throughout, none of the global message notifications are currently being vocalized by any screen reader software.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**

1. Open any Screen Reader (VO, NVDA, JAWS).
2. Go to the Sign-In page
3. Introduce a valid email address and an invalid password.
4. See error alert on top, and hear the absence of vocalization of that alert.

**Expected behavior**
- All Global message notifications get vocalized as soon as they happen.
- As some notifications require immediate user attention and others don't, please assign as follows:
- MSG_TYPE_CONFIRMATION use `aria-live="polite" aria-atomic="true"`
- MSG_TYPE_INFO use `aria-live="polite" aria-atomic="true"`
- MSG_TYPE_WARNING use `aria-live="assertive" aria-atomic="true"`
- MSG_TYPE_ERROR use `aria-live="assertive" aria-atomic="true"`

**Screenshots**

![Screen Shot 2021-09-09 at 8 39 49 PM](https://user-images.githubusercontent.com/15113219/132780781-6be7dbe9-1b13-4001-a68c-eb10b6e234b2.png)

**Example of potential solution**

![Screen Shot 2021-09-09 at 8 10 50 PM](https://user-images.githubusercontent.com/15113219/132780815-22cccaf1-5cf1-4f95-a704-491fe8d54df0.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using yarn
More extensive testing (on other screenreaders and devices) will be done with RC.

<img width="1343" alt="Screen Shot 2022-03-30 at 1 48 20 PM" src="https://user-images.githubusercontent.com/45409515/160899082-7be9be9e-c20c-4341-bb86-e726428b391d.png">
